domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/searchaliasfield/SearchAliasFieldTypeValidator.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.searchaliasfield;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.repository.SearchAliasFieldRepository;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.SearchAliasFieldEntity;
+
+@Component
+public class SearchAliasFieldTypeValidator implements SearchAliasFieldValidator {
+
+    private final SearchAliasFieldRepository repository;
+
+    @Autowired
+    public SearchAliasFieldTypeValidator(SearchAliasFieldRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public ValidationResult validate(SearchAliasFieldEntity searchAliasField) {
+
+        ValidationResult validationResult = new ValidationResult();
+
+        if (searchAliasField.getFieldType() == null) {
```

Comment:
just for safety shall we use 
StringUtils.isBlank(searchAliasField.getFieldType()) 
- Checks if a String is whitespace, empty ("") or null.
?

Reply:
getFieldType returns object FieldTypeEntity, if its null, no field type has been set. So thought a null check should suffice here.